addons/binding/org.openhab.binding.nest/ESH-INF/binding/binding.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="nest"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:binding="http://eclipse.org/smarthome/schemas/binding/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/binding/v1.0.0 http://eclipse.org/smarthome/schemas/binding-1.0.0.xsd">
+
+    <name>Nest Binding</name>
+    <description>Nest connects to the nest cloud and allows control of the various nest devices.  Create a bridge thing
```

Comment:
Ok...  ALthough I found this bit super confusing when I was trying to setup other bindings in openhab.  Took me a bit to figure out that what I was supposed to add was a bridge.  I guess I could put this in the docs part though.